IdentityCore/src/network/error_handler/MSIDAADRequestErrorHandler.h

```diff
@@ -26,7 +26,4 @@
 
 @interface MSIDAADRequestErrorHandler : NSObject <MSIDHttpRequestErrorHandling>
 
-@property (nonatomic) NSInteger retryCounter;
-@property (nonatomic) NSTimeInterval retryInterval;
-
```

Comment:
These two lines have been removed but seems added back accidentally by this PR:
https://github.com/AzureAD/microsoft-authentication-library-common-for-objc/commit/c087868f1c23c7347f24bf4ac4d2bfa7cc67cbf5#diff-31bccab38d271abf0c141fbf8856d8de

Reply:
yeah, those lines were added by mistake